lib/Controller/CallController.php

```diff
@@ -53,10 +53,11 @@ public function __construct(string $appName,
 	 * @return DataResponse
 	 */
 	public function getPeersForCall(): DataResponse {
+		$timeout = $this->timeFactory->getTime() - 30;
 		$result = [];
-		$participants = $this->room->getParticipants($this->timeFactory->getTime() - 30);
+		$participants = $this->room->getParticipantsInCall();
 		foreach ($participants as $participant) {
-			if ($participant->getSessionId() === '0' || $participant->getInCallFlags() === Participant::FLAG_DISCONNECTED) {
+			if ($participant->getLastPing() < $timeout) {
```

Comment:
Previously the participants were only taken into account if their last ping was [greater than timeout](https://github.com/nextcloud/spreed/blob/98ddbcd843ca8c1cb3a130385b21db27ff82a111/lib/Room.php#L984), but now they are taken into account if they are greater or equal. I am not sure if that could cause any issue with a check somewhere else.

Reply:
well one second less or not is not the problem I guess.